test/unit/calculators/coronavirus_find_support_calculator_test.rb

```diff
@@ -0,0 +1,534 @@
+require_relative "../../test_helper"
+
+module SmartAnswer::Calculators
+  class CoronavirusFindSupportCalculatorTest < ActiveSupport::TestCase
+    setup do
+      @calculator = CoronavirusFindSupportCalculator.new
+    end
+
+    context "#needs_help_with?" do
+      should "return true if the given help item has been chosen" do
+        @calculator.need_help_with = "one,two,three,four"
+        assert_equal @calculator.needs_help_with?("one"), true
+      end
+
+      should "return false if the given help item has not been chosen" do
+        @calculator.need_help_with = "one,two,three,four"
+        assert_equal @calculator.needs_help_with?("five"), false
+      end
+
+      should "return false if need_help_with is an empty string" do
+        @calculator.need_help_with = ""
+        assert_equal @calculator.needs_help_with?("one"), false
+      end
+
+      should "return false if need_help_with is nil" do
+        @calculator.need_help_with = nil
+        assert_equal @calculator.needs_help_with?("one"), false
+      end
+    end
+
+    context "#has_results?" do
+      should "return true if at least one help item has been chosen" do
+        @calculator.need_help_with = "one"
+        assert_equal @calculator.has_results?, true
+      end
+
+      should "return false if need_help_with is an empty string" do
+        @calculator.need_help_with = ""
+        assert_equal @calculator.has_results?, false
+      end
+
+      should "return false if need_help_with is nil" do
+        @calculator.need_help_with = nil
+        assert_equal @calculator.has_results?, false
+      end
+    end
+
+    context "#needs_help_in?" do
+      should "return true if the given nation has been chosen" do
+        @calculator.nation = "one,two,three,four"
+        assert_equal @calculator.needs_help_in?("one"), true
+      end
+
+      should "return false if the given nation has not been chosen" do
+        @calculator.nation = "one,two,three,four"
+        assert_equal @calculator.needs_help_in?("five"), false
+      end
+
+      should "return false if nation is an empty string" do
+        @calculator.nation = ""
+        assert_equal @calculator.needs_help_in?("one"), false
+      end
+
+      should "return false if nation is nil" do
+        @calculator.nation = nil
+        assert_equal @calculator.needs_help_in?("one"), false
+      end
+    end
+
+    context "#user_feels_unsafe?" do
+      context "user has selected feeling_unsafe" do
```

Comment:
I think this should be:
```ruby
    describe "#user_feels_unsafe?" do
      context "user has selected feeling_unsafe" do
```
The outer block is describing the behaviour of the method, and the inner block wraps a section where that behaviour is altered by an external (to the method) change to the context within which the method is being called.

Reply:
Thanks @reggieb - I agree, it should be `describe`.  However, `describe` throws an error!

Reply:
That's weird. The minitest documentation suggests it should work. https://github.com/seattlerb/minitest#specs-

Reply:
What happens if you add `require 'minitest/spec'` to the start of the file?

Reply:
Either I get `undefined method 'assert_current_node'` or `undefined method 'need_help_with='` (depending on the spec file).  Looks like the issue is a bunch of `include`s have been added to `ActiveSupport::TestCase`.  I don't believe it's a valuable use of time to fix this, given it's just a semantic difference between `context` and `describe`.

Reply:
I agree